app/controllers/api/v2/hosts_controller.rb

```diff
@@ -82,7 +82,7 @@ def show
           end
           param :puppetclass_ids, Array
           param :config_group_ids, Array, :desc => N_("IDs of associated config groups")
-          param :operatingsystem_id, String, :desc => N_("required if host is managed and value is not inherited from host group")
+          param :operatingsystem_id, :number, :desc => N_("required if host is managed and value is not inherited from host group")
           param :medium_id, String, :desc => N_("required if not imaged based provisioning and host is managed and value is not inherited from host group")
```

Comment:
Should we do it globaly everywhere?
```suggestion
          param :medium_id, :number, :desc => N_("required if not imaged based provisioning and host is managed and value is not inherited from host group")
```

Reply:
that's what we discussed, the PR was changed to use `Refs` to avoid closing the issue that tracks changing it everywhere